sh.exe.stackdump

```diff
@@ -0,0 +1,9 @@
+Exception: STATUS_STACK_OVERFLOW at rip=7FFF037E4FD7
+rax=0000000000001250 rbx=00000000FFFFA970 rcx=0000000000000000
+rdx=0000000180010018 rsi=0000000180271780 rdi=000000018021C490
+r8 =00000000000B35D2 r9 =0000000180271780 r10=00000000FFFF9000
+r11=00000000FFE030F0 r12=00000000FFFFA850 r13=000000018021C490
+r14=00000000FFFFA7F0 r15=00000000FFFFA978
+rbp=0000000000000420 rsp=00000000FFFFA658
+program=C:\Users\Vlad\AppData\Local\GitHubDesktop\app-1.1.1\resources\app\git\usr\bin\sh.exe, pid 24660, thread unknown (0x605C)
+cs=0033 ds=002B es=002B fs=0053 gs=002B ss=002B
```

Comment:
Could you please remove this file 😉

Reply:
What in the world?

Ok, I removed it. Sorry about that.